service/src/main/kotlin/fi/espoo/evaka/pis/Employee.kt

```diff
@@ -14,6 +14,7 @@ data class Employee(
     val lastName: String,
     val email: String?,
     val externalId: ExternalId?,
+    val employeeNumber: String?,
```

Comment:
Do we need the field here? As far as I understand, the employee number is not actually used anywhere except login and later in integration-related queries.

Reply:
Fixed